packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -899,7 +899,7 @@ export interface CallWithChatAdapterManagement {
     submitSurvey(survey: CallSurvey): Promise<CallSurveyResponse | undefined>;
     unmute(): Promise<void>;
     updateBackgroundPickerImages(backgroundImages: VideoBackgroundImage[]): void;
-    updateMessage(messageId: string, content: string, metadata?: Record<string, string>): Promise<void>;
+    updateMessage(messageId: string, content: string, options?: Record<string, string>): Promise<void>;
```

Comment:
Do we have a documentation / deprecation explanation anywhere for this change?

Reply:
yes 
<img width="1136" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/c9bd18ef-4553-4ec8-a8af-9b2b71f54bd0">